packages/eslint-config-react-app/index.js

```diff
@@ -22,13 +22,39 @@
 // We blacklist the globals that we deem potentially confusing.
 // To use them, explicitly reference them, e.g. `window.name` or `window.status`.
 const restrictedGlobals = require('confusing-browser-globals');
+const readPkgUp = require('read-pkg-up');
+
+let hasJestDom = false;
+let hasTestingLibrary = false;
+
+try {
+  const pkg = readPkgUp.sync({ normalize: true });
+  const allDeps = Object.keys({
+    ...pkg.peerDependencies,
+    ...pkg.devDependencies,
+    ...pkg.dependencies,
```

Comment:
are there any optionalDependencies? or does that not matter

Reply:
Doesn't matter here I think